peer.go

```diff
@@ -390,13 +390,7 @@ out:
 		//  * .(CommitmentUpdater)
 
 		case *lnwire.ErrorGeneric:
-			switch msg.Code {
```

Comment:
Nice catch, this bit of code was unnecessary due to similar logic already residing within the `fundingManager.handleErrorGeneric` method.